lib/common.js

```diff
@@ -72,5 +73,20 @@ function memoize (fn) {
   });
 }
 
-module.exports = {cleanApp, request, memoize};
+function storeId (countryCode) {
+  const countries = c.countries;
+  const markets = c.markets;
+  const defaultStore = '143441';
+  if (!countryCode) {
+    return defaultStore;
+  }
+  for (var i in countries) {
```

Comment:
I don't get why you need to have both country and market maps? Can't you just have one map with country code as key and storefront Id as value? That way the whole function would be reduced to something like

`return c.markets[countryCode.toLowerCase()] || defaultStore`

Also, since it used in the `similar` method, this function could go in that file instead of common.